isso/tests/test_comments.py

```diff
@@ -33,6 +33,7 @@ def setUp(self):
         conf.set("general", "dbpath", self.path)
         conf.set("guard", "enabled", "off")
         conf.set("hash", "algorithm", "none")
+        conf.set("general", "latest-enabled", "true")
```

Comment:
Please also add a short entry to docs/docs/configuration/server.rst about this new configuration option.